scripts/read_spectrum.R

```diff
@@ -11,9 +11,18 @@ prepare_hc_leapfrog_projp <- function(pjnz, params, pop_1){
   v$paed_cd4_dist <- dp_read_paed_cd4_dist(pjnz) / 100
 
   prog = dp_read_paed_cd4_prog(pjnz)
-  paed_cd4_prog <- array(c(prog[1,1:6],0), dim = 7, dimnames = list(cd4 = c('30plus', '26-30', '21-25', '16-20', '11-15', '5-10', '<5')))
+  paed_cd4_prog <- array(0, dim = c(7,2,2), dimnames = list(cd4 = c('30plus', '26-30', '21-25', '16-20', '11-15', '5-10', '<5'),
```

Comment:
Are these CD4 percent categories?  Maybe label the name as `list(cd4percent = ...` if so.

Also are 26-30, 21-25, etc. accurate? Vs. 25-29, 20-24, etc.

Reply:
changed name to cd4percent, 

This stratification is what is used in Spectrum